File: backend-module/src/main/java/com/campus/eneity/ClassInfo.java
package com.campus.eneity;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class ClassInfo {
    private int id;
    private String grade;
    private String className;
    private int headTeacherId;
    private int studentCount;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private boolean isDeleted;
}

File: frontend/src/types/classInfo.ts
export interface ClassInfo {
  id: number
  grade: string
  className: string
  headTeacherId: number
  studentCount: number
  createdAt: string
  updatedAt: string
  isDeleted: boolean
}

export interface ClassInfoForm {
  id?: number
  grade: string
  className: string
  headTeacherId: number | null
}
